bin/connect-distributed.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 # Licensed to the Apache Software Foundation (ASF) under one or more
 # contributor license agreements.  See the NOTICE file distributed with
-# this work for additional information regarding copyright ownership.
+# this work for additional information regarding kright ownership.
```

Comment:
Haha, not even sed, just some random editor typo. You managed to catch that just before I pushed the fix :)